nupic/frameworks/opf/clamodel.py

```diff
@@ -396,10 +396,9 @@ def run(self, inputRecord):
     # For temporal classification. Not used, and might not work anymore
     elif self._isClassificationModel():
       inferences = self._classificationCompute()
-
-    results.inferences.update(inferences)
-
-    inferences = self._anomalyCompute()
+    else:
```

Comment:
I'm pretty sure this changes the results. Before the final results.inferences would contain the anomalyCompute results in addition to results from one of the previous calls.